packages/packagers/js/src/JSPackager.js

```diff
@@ -121,17 +121,18 @@ export default new Packager({
         wrapped += ']';
 
         if (options.sourceMaps) {
+          let lineCount = countLines(output);
           let assetMap =
             maps[i] ??
             SourceMap.generateEmptyMap(
               path
                 .relative(options.projectRoot, asset.filePath)
                 .replace(/\\+/g, '/'),
               output,
-            );
+            ).toBuffer();
```

Comment:
Is it worth avoiding converting the map to a buffer and then immediately back again or is that fast enough? i.e. could the C++ read the underlying SourceMap object directly instead of converting between buffers?

Reply:
wait a minute, you're right, this is stupid, I can just do call that function on map and not create another sourcemap instance at all 🤦‍♂️